package.json

```
@@ -36,54 +36,54 @@
     "@commitlint/cli": "12.1.4",
     "@commitlint/config-angular": "12.1.4",
     "@istanbuljs/nyc-config-typescript": "1.0.1",
-    "@nestjs/common": "7.6.15",
-    "@nestjs/core": "7.6.15",
-    "@nestjs/testing": "7.6.15",
-    "@release-it/conventional-changelog": "2.0.1",
-    "@types/chai": "4.2.18",
+    "@nestjs/common": "7.6.18",
+    "@nestjs/core": "7.6.18",
+    "@nestjs/testing": "7.6.18",
+    "@release-it/conventional-changelog": "3.0.1",
+    "@types/chai": "4.2.21",
     "@types/chai-as-promised": "7.1.4",
-    "@types/mocha": "8.2.2",
-    "@types/node": "14.14.45",
-    "@types/sinon": "10.0.0",
+    "@types/mocha": "8.2.3",
+    "@types/node": "14.17.5",
+    "@types/sinon": "10.0.2",
     "@types/sinon-chai": "3.2.5",
-    "@typescript-eslint/eslint-plugin": "4.23.0",
-    "@typescript-eslint/parser": "4.23.0",
+    "@typescript-eslint/eslint-plugin": "4.28.3",
+    "@typescript-eslint/parser": "4.28.3",
     "chai": "4.3.4",
     "chai-as-promised": "7.1.1",
     "conventional-changelog-angular": "5.0.12",
-    "dotenv": "9.0.2",
-    "eslint": "7.26.0",
+    "dotenv": "10.0.0",
+    "eslint": "7.30.0",
     "eslint-config-prettier": "8.3.0",
-    "husky": "6.0.0",
+    "husky": "7.0.1",
     "lint-staged": "11.0.0",
-    "mocha": "8.4.0",
+    "mocha": "9.0.2",
     "nyc": "15.1.0",
     "pinst": "2.1.6",
-    "prettier": "2.3.0",
+    "prettier": "2.3.2",
     "proxyquire": "2.1.3",
     "reflect-metadata": "0.1.13",
-    "release-it": "14.6.2",
+    "release-it": "14.10.0",
     "rimraf": "3.0.2",
-    "rxjs": "7.0.1",
+    "rxjs": "7.2.0",
     "semver": "7.3.5",
-    "sinon": "10.0.0",
-    "sinon-chai": "3.6.0",
+    "sinon": "11.1.1",
+    "sinon-chai": "3.7.0",
     "source-map-support": "0.5.19",
-    "ts-node": "9.1.1",
+    "ts-node": "10.1.0",
     "ts-toolbelt": "9.6.0",
-    "tslib": "2.2.0",
-    "typescript": "4.2.4"
+    "tslib": "2.3.0",
+    "typescript": "4.3.5"
   },
   "peerDependencies": {
     "@nestjs/common": "^6.0.0 || ^7.0.0",
     "@nestjs/core": "^6.0.0 || ^7.0.0",
```

здесь наверное тоже надо апгрейднуть

```suggestion
    "@nestjs/common": "^6.0.0 || ^8.0.0",
    "@nestjs/core": "^6.0.0 || ^8.0.0",
```